Source/MediaInfo/Tag/File_VorbisCom.cpp

```diff
@@ -17,7 +17,7 @@
 //---------------------------------------------------------------------------
 
 //---------------------------------------------------------------------------
-#if defined(MEDIAINFO_VORBISCOM_YES) || defined(MEDIAINFO_OGG_YES)
+#if defined(MEDIAINFO_TAG_YES) && (defined(MEDIAINFO_VORBISCOM_YES) || defined(MEDIAINFO_OGG_YES))

```

Comment:
Too much complex for no reason.
Replace by:

```
#if defined(MEDIAINFO_VORBISCOM_YES)
```

it is doable now thanks to your 2nd commit.

Reply:
That's even better, made it so - thanks for the hint!